alias.c

```diff
@@ -83,20 +83,29 @@ static void expand_aliases_r(struct AddressList *al, struct ListHead *expn)
           }
         }
 
-        if (!i)
+        if (i)
```

Comment:
Can we rename `i` to be more descriptive of it's purpose? I think `duplicate` makes the conditional easier to understand.
```C
if (duplicate)
{
            // We've already seen this alias, so drop it
          struct Address *next = TAILQ_NEXT(a, entries);
          TAILQ_REMOVE(al, a, entries);
          mutt_addr_free(&a);
          a = next;
          continue;
}
```
vs.
```C
if (i)
{
            // We've already seen this alias, so drop it
          struct Address *next = TAILQ_NEXT(a, entries);
          TAILQ_REMOVE(al, a, entries);
          mutt_addr_free(&a);
          a = next;
          continue;
}
```